src/models/applicationState.ts

```diff
@@ -220,16 +220,10 @@ export interface ILabelData {
  */
 export interface ILabel {
```

Comment:
This isn't used anywhere right? Why do we need it?

Reply:
It is used in ILabelData